lms/static/sass/base/_layouts.scss

```diff
@@ -0,0 +1,104 @@
+// base layout styles to support early responsive lms
+// may be discarded later once sass breakpoints are wired in
+
+body.view-incourse {
```

Comment:
NIT: do you need the ``body`` portion of this selector? Can we get away with just the ``.view-incourse``?